bmv2/Makefile.am

```diff
@@ -29,6 +29,35 @@ $(builddir)/@P4_NAME@/thrift-src/pd_rpc_server.h
 
 P4_JSON_OUTPUT = $(P4_NAME).json
 
+plugin_files =
+plugin_headers =
+
+# Openflow plugin
+if COND_PLUGIN_OPENFLOW
+of_sources = \
+@P4_NAME@/plugin/of/src/pd_wrappers.c    \
+@P4_NAME@/plugin/of/src/ofpat_groups.c   \
+@P4_NAME@/plugin/of/src/ofpat_state.c    \
+@P4_NAME@/plugin/of/src/ofpat_pipeline.c
+
+of_includes = \
+@P4_NAME@/plugin/of/inc/pd_wrappers.h    \
+@P4_NAME@/plugin/of/inc/ofpat_groups.h   \
+@P4_NAME@/plugin/of/src/ofpat_state.h    \
+@P4_NAME@/plugin/of/inc/ofpat_pipeline.h \
+@P4_NAME@/plugin/of/inc/openflow.h
+
+plugin_files += $(of_sources) $(of_includes)
+plugin_headers += $(of_includes)
```

Comment:
Btw, I am saying that because I think these headers are needed by p4ofagent, and therefore should be installed. If I'm wrong let me know.